components/ServiceCard/index.js

```diff
@@ -43,20 +46,24 @@ export default function ServiceCard({ id, user, isBooked }) {
   };
 
   if (isLoading || !service) {
-    return <H2>Loading...</H2>;
+    return <StyledHeadlineTwo>Loading...</StyledHeadlineTwo>;
   }
 
   if (error) {
     return (
-      <H2>
+      <StyledHeadlineTwo>
         There was an error fetching the service details. Please try again.
-      </H2>
+      </StyledHeadlineTwo>
     );
   }
 
   return (
     <>
-      {isBooked ? <H2>My Booking</H2> : <H2>Service</H2>}
+      {isBooked ? (
+        <StyledHeadlineTwo>My Booking</StyledHeadlineTwo>
+      ) : (
+        <StyledHeadlineTwo>Service</StyledHeadlineTwo>
+      )}
```

Comment:
You can use a more concise approach by conditionally setting the content of the `<StyledHeadlineTwo> `component. This approach maintains the same logic but minimizes repetition by only conditionally changing the content within the component rather than the entire component.

```suggestion
      {isBooked ? (
     <StyledHeadlineTwo>{isBooked ? 'My Booking' : 'Service'}</StyledHeadlineTwo>
      )}
```